File: Python Excel Examples/WorksheetsApiDemo/comments/updateComment.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration

appId = "your id"
appKey = "your key"
baseUrl="https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.worksheets_api.WorksheetsApi(configuration)

name = "UpdateComment.xlsx"
storage = ""
folder = "ExcelDocument"
sheet_name = "Sheet1"
cellName = "C6"
comment = spirecloudexcel.models.Comment()
comment.note = "Hello,Comments"
comment.width = 160
comment.height = 30
comment.is_visible = True
comment.text_horizontal_alignment = "center"
comment.auto_size = True
api.update_comment(name, sheet_name, cellName, comment, storage=storage, folder=folder)

File: JavaScript Excel Examples/WorksheetsApiDemo/Comments/updateComment.js
(function (){
    var appId = "your id";
    var appKey = "your key";
    var baseUrl = "https://api.e-iceblue.cn";

    var SpirecloudExcel = require('../../../src/index');
    var apiClient = new SpirecloudExcel.ApiClient(appId, appKey,baseUrl);
    var instance = new SpirecloudExcel.WorksheetsApi(apiClient);
    
    var name = "updateComment.xlsx";
    var opts = { folder: "input" };
    var sheetName = "Sheet1";
    var cellName = "F9";
    var comment = new SpirecloudExcel.Comment();
    comment.Author = "jerry";
    comment.Note = "Hello,Comments";
    comment.TextOrientationType = "TopToBottom";
    comment.IsVisible = false;
    comment.AutoSize = true;
    instance.updateComment(name, sheetName, cellName, comment, opts,
            function (error, data, response) {
        if (error) throw error;
    });
})();